2019.05.14 14:53:19 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:53:19 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:53:19 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:53:19 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:53:19 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:53:19 LOG5[ui]: FIPS mode disabled
2019.05.14 14:53:19 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:53:19 LOG5[ui]: Configuration successful
2019.05.14 14:53:55 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17491
2019.05.14 14:54:00 LOG3[0]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:54:00 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:54:53 LOG5[ui]: Terminated
2019.05.14 14:54:55 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:54:55 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:54:55 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:54:55 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:54:55 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:54:55 LOG5[ui]: FIPS mode disabled
2019.05.14 14:54:55 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:54:55 LOG5[ui]: Configuration successful
2019.05.14 14:55:07 LOG5[0]: Service [https] accepted connection from 183.192.90.7:17529
2019.05.14 14:55:10 LOG3[0]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:55:10 LOG5[0]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:55:18 LOG5[1]: Service [https] accepted connection from 183.192.90.7:17537
2019.05.14 14:55:19 LOG3[1]: SSL_accept: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2019.05.14 14:55:19 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:55:27 LOG5[2]: Service [https] accepted connection from 183.192.90.7:17545
2019.05.14 14:55:28 LOG5[3]: Service [https] accepted connection from 183.192.90.7:17546
2019.05.14 14:55:28 LOG3[2]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 14:55:28 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:28 LOG5[2]: Service [https] connected remote server from 127.0.0.1:53972
2019.05.14 14:55:28 LOG3[3]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 14:55:28 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:28 LOG5[3]: Service [https] connected remote server from 127.0.0.1:53976
2019.05.14 14:55:29 LOG5[4]: Service [https] accepted connection from 183.192.90.7:17548
2019.05.14 14:55:32 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.14 14:55:32 LOG5[4]: Service [https] connected remote server from 127.0.0.1:53980
<QueryDict: {}>
Performing system checks...

System check identified no issues (0 silenced).
May 14, 2019 - 22:54:59
Django version 2.2.1, using settings 'vote.settings'
Starting development server at http://0.0.0.0:8888/
Quit the server with CONTROL-C.
2019.05.14 14:57:42 LOG3[4]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[4]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 14:57:42 LOG3[3]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[3]: Connection closed: 266 byte(s) sent to TLS, 306 byte(s) sent to socket
2019.05.14 14:57:42 LOG3[2]: transfer: s_poll_wait: TIMEOUTclose exceeded: closing
2019.05.14 14:57:42 LOG5[2]: Connection closed: 748 byte(s) sent to TLS, 1273 byte(s) sent to socket
2019.05.14 14:57:42 LOG5[ui]: Terminated
2019.05.14 14:57:45 LOG5[ui]: stunnel 5.44 on x86_64-redhat-linux-gnu platform
2019.05.14 14:57:45 LOG5[ui]: Compiled/running with OpenSSL 1.0.2k-fips  26 Jan 2017
2019.05.14 14:57:45 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2019.05.14 14:57:45 LOG5[ui]: Reading configuration from file /root/mp-vote/backend/stunnel/dev_https
2019.05.14 14:57:45 LOG5[ui]: UTF-8 byte order mark not detected
2019.05.14 14:57:45 LOG5[ui]: FIPS mode disabled
2019.05.14 14:57:45 LOG4[ui]: Insecure file permissions on stunnel/private.key
2019.05.14 14:57:45 LOG5[ui]: Configuration successful
2019.05.14 15:11:28 LOG5[0]: Service [https] accepted connection from 183.192.90.7:18471
2019.05.14 15:11:29 LOG5[0]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:11:29 LOG5[0]: Service [https] connected remote server from 127.0.0.1:54600
2019.05.14 15:11:32 LOG5[0]: Connection closed: 266 byte(s) sent to TLS, 385 byte(s) sent to socket
2019.05.14 15:12:28 LOG5[1]: Service [https] accepted connection from 61.151.178.177:42841
2019.05.14 15:12:28 LOG3[1]: SSL_accept: Peer suddenly disconnected
2019.05.14 15:12:28 LOG5[1]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 15:18:11 LOG5[2]: Service [https] accepted connection from 183.192.90.7:18882
2019.05.14 15:18:11 LOG5[2]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:18:11 LOG5[2]: Service [https] connected remote server from 127.0.0.1:54780
2019.05.14 15:18:24 LOG5[2]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 15:19:48 LOG5[3]: Service [https] accepted connection from 183.192.172.11:19498
2019.05.14 15:19:50 LOG3[3]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 15:19:50 LOG5[3]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:19:50 LOG5[3]: Service [https] connected remote server from 127.0.0.1:54828
2019.05.14 15:19:51 LOG5[3]: Connection closed: 266 byte(s) sent to TLS, 152 byte(s) sent to socket
2019.05.14 15:20:24 LOG5[4]: Service [https] accepted connection from 183.192.172.11:19500
2019.05.14 15:20:25 LOG5[4]: s_connect: connected 127.0.0.1:8888
2019.05.14 15:20:25 LOG5[4]: Service [https] connected remote server from 127.0.0.1:54842
2019.05.14 15:20:25 LOG5[4]: Connection closed: 266 byte(s) sent to TLS, 152 byte(s) sent to socket
2019.05.14 17:15:30 LOG5[5]: Service [https] accepted connection from 122.228.19.79:29614
2019.05.14 17:15:30 LOG3[5]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 17:15:30 LOG5[5]: s_connect: connected 127.0.0.1:8888
2019.05.14 17:15:30 LOG5[5]: Service [https] connected remote server from 127.0.0.1:58040
2019.05.14 17:15:31 LOG5[5]: Connection closed: 266 byte(s) sent to TLS, 177 byte(s) sent to socket
2019.05.14 20:26:10 LOG5[6]: Service [https] accepted connection from 107.170.200.205:59360
2019.05.14 20:26:12 LOG3[6]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.14 20:26:12 LOG5[6]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 20:57:50 LOG5[7]: Service [https] accepted connection from 195.154.61.206:43565
2019.05.14 20:57:51 LOG3[7]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.14 20:57:51 LOG5[7]: s_connect: connected 127.0.0.1:8888
2019.05.14 20:57:51 LOG5[7]: Service [https] connected remote server from 127.0.0.1:35616
2019.05.14 20:58:04 LOG5[7]: Connection closed: 266 byte(s) sent to TLS, 238 byte(s) sent to socket
2019.05.14 21:18:32 LOG5[8]: Service [https] accepted connection from 162.243.151.182:52342
2019.05.14 21:18:32 LOG5[8]: s_connect: connected 127.0.0.1:8888
2019.05.14 21:18:32 LOG5[8]: Service [https] connected remote server from 127.0.0.1:36158
2019.05.14 21:18:32 LOG5[8]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.14 23:45:46 LOG5[9]: Service [https] accepted connection from 196.52.43.90:56319
2019.05.14 23:45:48 LOG3[9]: SSL_accept: Peer suddenly disconnected
2019.05.14 23:45:48 LOG5[9]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:53:16 LOG5[10]: Service [https] accepted connection from 184.105.139.68:51774
2019.05.15 00:53:19 LOG5[10]: s_connect: connected 127.0.0.1:8888
2019.05.15 00:53:19 LOG5[10]: Service [https] connected remote server from 127.0.0.1:41780
2019.05.15 00:53:20 LOG5[10]: Connection closed: 266 byte(s) sent to TLS, 40 byte(s) sent to socket
2019.05.15 00:54:54 LOG5[11]: Service [https] accepted connection from 184.105.139.68:9700
2019.05.15 00:54:54 LOG3[11]: SSL_accept: 1408A10B: error:1408A10B:SSL routines:ssl3_get_client_hello:wrong version number
2019.05.15 00:54:54 LOG5[11]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:55:20 LOG5[12]: Service [https] accepted connection from 184.105.139.68:6962
2019.05.15 00:55:20 LOG3[12]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.15 00:55:20 LOG5[12]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 00:55:44 LOG5[13]: Service [https] accepted connection from 184.105.139.68:4214
2019.05.15 00:55:45 LOG3[13]: SSL_accept: 1408A0C1: error:1408A0C1:SSL routines:ssl3_get_client_hello:no shared cipher
2019.05.15 00:55:45 LOG5[13]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 01:19:00 LOG5[14]: Service [https] accepted connection from 104.248.85.56:39922
2019.05.15 01:19:00 LOG3[14]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 01:19:00 LOG5[14]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 01:32:40 LOG5[15]: Service [https] accepted connection from 71.6.232.7:46034
2019.05.15 01:32:41 LOG3[15]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 01:32:41 LOG5[15]: s_connect: connected 127.0.0.1:8888
2019.05.15 01:32:41 LOG5[15]: Service [https] connected remote server from 127.0.0.1:42808
2019.05.15 01:32:48 LOG5[15]: Connection closed: 266 byte(s) sent to TLS, 205 byte(s) sent to socket
2019.05.15 03:26:05 LOG5[16]: Service [https] accepted connection from 131.153.31.59:51751
2019.05.15 03:26:05 LOG5[16]: s_connect: connected 127.0.0.1:8888
2019.05.15 03:26:05 LOG5[16]: Service [https] connected remote server from 127.0.0.1:45756
2019.05.15 03:26:05 LOG5[16]: Connection closed: 266 byte(s) sent to TLS, 78 byte(s) sent to socket
2019.05.15 05:10:59 LOG5[17]: Service [https] accepted connection from 198.108.66.240:5572
2019.05.15 05:10:59 LOG3[17]: SSL_accept: 1408A10B: error:1408A10B:SSL routines:ssl3_get_client_hello:wrong version number
2019.05.15 05:10:59 LOG5[17]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 06:36:36 LOG5[18]: Service [https] accepted connection from 176.58.124.134:57266
2019.05.15 06:36:36 LOG3[18]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 06:36:36 LOG5[18]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 07:02:25 LOG5[19]: Service [https] accepted connection from 164.52.24.162:49001
2019.05.15 07:02:25 LOG3[19]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 07:02:25 LOG5[19]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 07:16:05 LOG5[20]: Service [https] accepted connection from 139.162.113.204:39724
2019.05.15 07:16:05 LOG5[20]: s_connect: connected 127.0.0.1:8888
2019.05.15 07:16:05 LOG5[20]: Service [https] connected remote server from 127.0.0.1:51818
2019.05.15 07:16:06 LOG5[20]: Connection closed: 266 byte(s) sent to TLS, 122 byte(s) sent to socket
2019.05.15 09:04:04 LOG5[21]: Service [https] accepted connection from 178.128.194.144:44324
2019.05.15 09:04:05 LOG3[21]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 09:04:05 LOG5[21]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:04:05 LOG5[21]: Service [https] connected remote server from 127.0.0.1:54648
2019.05.15 09:04:05 LOG5[21]: Connection closed: 266 byte(s) sent to TLS, 207 byte(s) sent to socket
2019.05.15 09:04:05 LOG5[22]: Service [https] accepted connection from 178.128.194.144:44612
2019.05.15 09:04:05 LOG3[22]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 09:04:05 LOG5[22]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 09:04:05 LOG5[23]: Service [https] accepted connection from 178.128.194.144:44756
2019.05.15 09:04:05 LOG3[23]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 09:04:05 LOG5[23]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 09:24:04 LOG5[24]: Service [https] accepted connection from 124.207.119.80:50436
2019.05.15 09:24:05 LOG5[24]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:24:05 LOG5[24]: Service [https] connected remote server from 127.0.0.1:55168
2019.05.15 09:24:14 LOG5[24]: Connection closed: 266 byte(s) sent to TLS, 111 byte(s) sent to socket
2019.05.15 09:24:34 LOG5[25]: Service [https] accepted connection from 35.246.247.9:53824
2019.05.15 09:24:34 LOG3[25]: s_connect: connect ::1:8888: Connection refused (111)
2019.05.15 09:24:34 LOG5[25]: s_connect: connected 127.0.0.1:8888
2019.05.15 09:24:34 LOG5[25]: Service [https] connected remote server from 127.0.0.1:55186
2019.05.15 09:24:34 LOG5[25]: Connection closed: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 10:53:58 LOG5[26]: Service [https] accepted connection from 106.75.3.52:53780
2019.05.15 10:53:58 LOG3[26]: SSL_accept: 1407609C: error:1407609C:SSL routines:SSL23_GET_CLIENT_HELLO:http request
2019.05.15 10:53:58 LOG5[26]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 10:54:04 LOG5[27]: Service [https] accepted connection from 106.75.3.52:57382
2019.05.15 10:54:04 LOG3[27]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 10:54:04 LOG5[27]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:03:10 LOG5[28]: Service [https] accepted connection from 108.178.16.154:39392
2019.05.15 11:03:10 LOG3[28]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:03:10 LOG5[28]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:03:13 LOG5[29]: Service [https] accepted connection from 108.178.16.154:41488
2019.05.15 11:03:13 LOG3[29]: SSL_accept: 1408A0A0: error:1408A0A0:SSL routines:ssl3_get_client_hello:length too short
2019.05.15 11:03:13 LOG5[29]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG3[30]: getpeerbyname (local_rfd): Transport endpoint is not connected (107)
2019.05.15 11:17:45 LOG5[30]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG5[31]: Service [https] accepted connection from 92.119.160.11:22347
2019.05.15 11:17:45 LOG3[31]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:17:45 LOG5[31]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:17:45 LOG5[32]: Service [https] accepted connection from 92.119.160.11:22859
2019.05.15 11:17:45 LOG3[32]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:17:45 LOG5[32]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
----------------------------------------
Exception happened during processing of request from ('43.249.192.59', 3469)
Traceback (most recent call last):
  File "/usr/lib64/python3.6/socketserver.py", line 651, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib64/python3.6/socketserver.py", line 361, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib64/python3.6/socketserver.py", line 721, in __init__
    self.handle()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 171, in handle
    self.handle_one_request()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 179, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib64/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------
----------------------------------------
Exception happened during processing of request from ('43.249.192.59', 1488)
Traceback (most recent call last):
  File "/usr/lib64/python3.6/socketserver.py", line 651, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib64/python3.6/socketserver.py", line 361, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib64/python3.6/socketserver.py", line 721, in __init__
    self.handle()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 171, in handle
    self.handle_one_request()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 179, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib64/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------
----------------------------------------
Exception happened during processing of request from ('110.249.212.46', 44868)
Traceback (most recent call last):
  File "/usr/lib64/python3.6/socketserver.py", line 651, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib64/python3.6/socketserver.py", line 361, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib64/python3.6/socketserver.py", line 721, in __init__
    self.handle()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 171, in handle
    self.handle_one_request()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 179, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib64/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------
----------------------------------------
Exception happened during processing of request from ('185.156.177.9', 4945)
Traceback (most recent call last):
  File "/usr/lib64/python3.6/socketserver.py", line 651, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib64/python3.6/socketserver.py", line 361, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib64/python3.6/socketserver.py", line 721, in __init__
    self.handle()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 169, in handle
    self.handle_one_request()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/servers/basehttp.py", line 179, in handle_one_request
    self.raw_requestline = self.rfile.readline(65537)
  File "/usr/lib64/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 104] Connection reset by peer
----------------------------------------
Performing system checks...

System check identified no issues (0 silenced).
May 14, 2019 - 22:57:48
Django version 2.2.1, using settings 'vote.settings'
Starting development server at http://0.0.0.0:8888/
Quit the server with CONTROL-C.
2019.05.15 11:45:56 LOG5[33]: Service [https] accepted connection from 185.10.68.172:45728
2019.05.15 11:45:56 LOG3[33]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:45:56 LOG5[33]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
2019.05.15 11:45:56 LOG5[34]: Service [https] accepted connection from 185.10.68.172:45876
2019.05.15 11:45:56 LOG3[34]: SSL_accept: 140760FC: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
2019.05.15 11:45:56 LOG5[34]: Connection reset: 0 byte(s) sent to TLS, 0 byte(s) sent to socket
Performing system checks...

System check identified no issues (0 silenced).
May 15, 2019 - 19:33:32
Django version 2.2.1, using settings 'vote.settings'
Starting development server at http://0.0.0.0:8888/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
May 15, 2019 - 19:46:26
Django version 2.2.1, using settings 'vote.settings'
Starting development server at http://0.0.0.0:8888/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
May 15, 2019 - 19:47:51
Django version 2.2.1, using settings 'vote.settings'
Starting development server at http://0.0.0.0:8888/
Quit the server with CONTROL-C.
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib64/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/utils/autoreload.py", line 54, in wrapper
    fn(*args, **kwargs)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 117, in inner_run
    self.check(display_num_errors=True)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/urls/resolvers.py", line 398, in check
    for pattern in self.url_patterns:
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/urls/resolvers.py", line 579, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib/python3.6/site-packages/django/urls/resolvers.py", line 572, in urlconf_module
    return import_module(self.urlconf_name)
  File "/root/.local/share/virtualenvs/backend-Q4ZaDRC0/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/mp-vote/backend/vote/vote/urls.py", line 53
    if request.method == 'GET':
     ^
IndentationError: expected an indented block

Performing system checks...

[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
Traceback (most recent call last):
  File "./vote/manage.py", line 21, in <module>
    main()
  File "./vote/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/base.py", line 361, in execute
    self.check()
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/urls/resolvers.py", line 398, in check
    for pattern in self.url_patterns:
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/urls/resolvers.py", line 579, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/site-packages/django/urls/resolvers.py", line 572, in urlconf_module
    return import_module(self.urlconf_name)
  File "/home/kevin/.local/share/virtualenvs/backend-Q_IvTKKv/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kevin/Projects/mp-vote/backend/vote/vote/urls.py", line 37, in <module>
    from weixin.login import WeixinLogin
ModuleNotFoundError: No module named 'weixin.login'
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
[ ] Clients allowed=500
[.] stunnel 5.51 on x86_64-pc-linux-gnu platform
[.] Compiled/running with OpenSSL 1.1.1b  26 Feb 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI
[ ] errno: (*__errno_location ())
[.] Reading configuration from file /home/kevin/Projects/mp-vote/backend/stunnel/dev_https
[.] UTF-8 byte order mark not detected
[.] FIPS mode disabled
[ ] Compression disabled
[ ] No PRNG seeding was required
[ ] Initializing service [https]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] Loading certificate from file: stunnel/certificate.crt
[ ] Certificate loaded from file: stunnel/certificate.crt
[ ] Loading private key from file: stunnel/private.key
[:] Insecure file permissions on stunnel/private.key
[ ] Private key loaded from file: stunnel/private.key
[ ] Private key check succeeded
[ ] DH initialization not needed
[ ] ECDH initialization
[ ] ECDH initialized with curves X25519:P-256:X448:P-521:P-384
[.] Configuration successful
[ ] Binding service [https]
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to 0.0.0.0:443: Permission denied (13)
[ ] Listening file descriptor created (FD=7)
[ ] Setting accept socket options (FD=7)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [https] to :::443: Permission denied (13)
[!] Binding service [https] failed
[ ] Deallocating section defaults
[ ] Unbinding service [https]
[ ] Service [https] closed
[ ] Deallocating section [https]
